practice-app/practice/cron.py

```diff
@@ -0,0 +1,13 @@
+from django_cron import CronJobBase, Schedule
+import requests
+
+class GetExchangeRates(CronJobBase):
+    RUN_EVERY_MINS = 1
+
+    schedule = Schedule(run_every_mins=RUN_EVERY_MINS)
+    code = 'practice.GetExchangeRates'
+
+    def do(self):
+        response = str(requests.request('GET', 'https://api.exchangeratesapi.io/latest').json())
+        with open('exchangeRates.txt', 'a') as log_file:
```

Comment:
I think instead of saving as a txt file, we would prefer to use Parity model and save the data to the db. It has all the necessary fields. You can check [here](https://github.com/bounswe/bounswe2019group2/blob/3ea289ff2874f4ed88d5a57b02063e07ce937477/practice-app/api/models.py#L57).

Reply:
Thanks for the feedback @mertyg. I was just experimenting :) This will be my first pull request. :blush

Reply:
I think it's really good to push the work you have done immediately. Keep pushing as you progress. That way we can interactively examine the progress you are making.
I also think that saving the response into a file is a good practice while experimenting but it would be better if you didn't commit the file. If we commit this kind of data files into git, the repository will grow quickly.

Reply:
Then, I will close this pull request and I will open a new one.

Reply:
No you dont need to, the file is already in the history there is no coming back from that. I just said it for the future. Wasn't that important anyways.